packages/web-app-files/src/views/Trashbin.vue

```diff
@@ -127,13 +127,7 @@ export default {
       this.loading = true
       this.CLEAR_CURRENT_FILES_LIST()
 
-      const resources = await this.$client.fileTrash.list('', '1', [
-        '{http://owncloud.org/ns}trashbin-original-filename',
-        '{http://owncloud.org/ns}trashbin-original-location',
-        '{http://owncloud.org/ns}trashbin-delete-datetime',
-        '{DAV:}getcontentlength',
-        '{DAV:}resourcetype'
-      ])
+      const resources = await this.$client.fileTrash.list('', '1', DavProperties.Trashbin)
```

Comment:
Why is this one only three strings now and was five before?

Reply:
The `DavProperties.Trashbin` is actually a lot more than the 5 properties, because we're also requesting the `DavProperties.Default` properties collection with it. Was thinking about keeping it at the bare minimum, but on the other hand the `Default` set contains quite useful data. What do you think? 🤔

Reply:
Hmmm true I wasn't aware at first. Let's discuss the implications tomorrow with a fresh mind, I'm a fan of communicating/fetching only what's needed

Reply:
Reduced it to the bare-minimum set for now (but within the constant).